examples/pytorch/graphsage/dist/train_dist.py

```diff
@@ -307,7 +309,7 @@ def main(args):
     parser.add_argument('--ip_config', type=str, help='The file for IP configuration')
     parser.add_argument('--part_config', type=str, help='The path to the partition config file')
     parser.add_argument('--num_clients', type=int, help='The number of clients')
-    parser.add_argument('--n_classes', type=int, help='the number of classes')
+    parser.add_argument('--n_classes', type=int, default=0, help='the number of classes')
```

Comment:
may be -1 is a better default value.

Please tell what the default value means and what is the impact in the help message.